### PR TITLE
API Update

### PR DESCRIPTION
**Pull request by mahipat99**
-Updated OTT API, EPG URL
-Added Expiry date in userDetails for debug

### CODE REVIEW

File: code_samples/utils.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def processTokenChunks(channelList):
                                                                             channel['channel_name']))
             print('Continuing...Please get license manually for channel :', channel['channel_name'])
         m3ustr += kodiPropLicenseType + "\n" + kodiPropLicenseUrl + "\n" + "#EXTINF:-1 "
-        m3ustr += "tvg-id=" + "\"" + "ts" + channel['channel_id'] + "\" " + "group-title=" + "\"" + channel['channel_genre'] + "\" " + "tvg-logo=\"" + channel[
```
Comment (ForceGT) [inline code_samples/utils.py:25]: This thing changes the compatibility for Kodi and OTT Nav right?
@Shra1V32 confirm please

**Comment by Shra1V32** [comment]
Dude, So many commits

**Comment by Shra1V32** [comment]
Merge it all into one commit, Then send PR again plox

**Comment by Shra1V32** [comment]
Has been merged with your authorship, Can't pull out the PR as this creates a spam in commits. Thank You.

**Pull request closed without merging**